assets/js/src/public.jsx

```diff
@@ -82,13 +91,16 @@ jQuery(($) => {
       delay = 0;
     }
     setTimeout(() => {
-      formDiv.addClass('active');
-      checkFormContainer(formDiv);
-
-      if (showOverlay) {
-        formDiv.prev('.mailpoet_form_popup_overlay').addClass('active');
-      }
+      doDisplayForm(formDiv, showOverlay);
     }, delay * 1000);
+
+    const exitIntentEnabled = form.data('exit-intent-enabled');
+    if (exitIntentEnabled) {
+      $(document).on('mouseleave', () => {
+        doDisplayForm(formDiv, showOverlay);
```

Comment:
I think we need to clear the timeout for displaying popup here otherwise the popup will be displayed again after the timeout.